tabular/src/autogluon/tabular/task/tabular_prediction/dataset.py

```diff
@@ -18,48 +17,29 @@ class TabularDataset(pd.DataFrame):
 
     Parameters
     ----------
-    file_path : str (optional)
-        Path to the data file (may be on local filesystem or URL to cloud s3 bucket). 
-        At least one of `file_path` and `df` arguments must be specified when constructing a new `TabularDataset`.
-    df : `pandas.DataFrame` (optional)
-        If you already have your data in a pandas Dataframe, you can directly provide it by specifying `df`. 
-        At least one of `file_path` and `df` arguments must be specified when constructing new `TabularDataset`.
-    feature_types : dict (optional)
-        Mapping from column_names to string describing data type of each column. 
-        If not specified, AutoGluon's fit() will automatically infer what type of data each feature contains.
-    subsample : int (optional)
-        If specified = k, we only keep first k rows of the provided dataset.
-    name : str (optional)
-        Optional name to assign to dataset (has no effect beyond being accessible via `TabularDataset.name`).
-    copy : bool (optional, default=False)
-        If True and `df` is passed, then `df` will be deep copied, resulting in two instances of the provided data in memory.
-        If False, the original `df` and the new `TabularDataset` will share the same underlying data.
-        Set `copy=True` if you don't want AutoGluon to be able to modify the original DataFrame during it's training process (and your machine has enough memory).
+    data : :class:`pd.DataFrame` or str
+        Path to data file (csv or parquet) or loaded :class:`pd.DataFrame`.
+        If you already have your data in a :class:`pd.DataFrame`, you can directly provide it by specifying `data`.
```

Comment:
might be more clearly documented as:

If string, path to data file (CSV or Parquet format).  Otherwise if your data are already in a pandas :class:`pd.DataFrame`, you can specify the DataFrame here.

Reply:
added